backend/src/main/resources/application.yaml

```diff
@@ -50,24 +50,9 @@ spring:
       initialize-schema: never
       table-name: ${spring.jpa.properties.hibernate.default_schema}.spring_session
 management:
-  health.probes.enabled: true
-graphql:
-  servlet:
-    mapping: /graphql
-    enabled: true
-    corsEnabled: true
-    exception-handlers-enabled: true
-  playground:
-    enabled: false
-  tools:
-    schema-location-pattern: "**/*.graphqls"
-  datetime:
-    scalars:
-      Date:
-        scalar-name: DateTime
-        format: yyyy-MM-dd'T'HH:MM:ss'Z'
-      localDate:
-        scalarName: MyLocalDate # graphql-datetime injects the LocalDate scalar no matter what. this rename allows us to reuse the LocalDate in FlexibleDateScalar
+  endpoint.health.probes.enabled: true
+  endpoint.info.enabled: true
+  endpoints.web.exposure.include: health, info
```

Comment:
the new spring version only exposes `health` endpoint by default, so I had to also expose the `info` endpoint since we use that in the pipeline
Here is the `PROD` health endpoint for reference, you should be able to access it without login
https://api-prod.simplereport.gov/actuator/info